index.qmd

```diff
@@ -12,22 +12,17 @@ It also helps R users extend their analyses by accessing the authoritative datas
 
 # Use ArcGIS Location Services 
 
-The **`arcgis`** meta-package for location services brings ArcGIS to data scientists, analysts, and developers where they work. At it's core, **`arcgis`** is a collection of R packages designed to interact with [ArcGIS Location Services](https://developers.arcgis.com/documentation/mapping-apis-and-services/) all from the comfort of R and your text editor. 
+The **`arcgis`** meta-package for location services brings ArcGIS to data scientists, analysts, and developers where they work. At its core, **`arcgis`** is a collection of R packages designed to interact with [ArcGIS Location Services](https://developers.arcgis.com/documentation/mapping-apis-and-services/) all from the comfort of R and your development environment of choice. 
 
-ArcGIS Location Services are provided as REST API endpoints and **`arcgis`** makes interacting with the services using R native objects seamless.
+ArcGIS Location Services are provided as REST API endpoints. **`arcgis`** enables seamless interaction with these services using R native objects. Examples of ArcGIS Location Services include:
 
-<!-- 
-Nick: Below this, suggest a bulleted list of a few of the location services, using the language of the developers doc:
-- Data hosting
-- Geocoding
-- Routing
+- [Data hosting](https://developers.arcgis.com/documentation/mapping-apis-and-services/data-hosting/)
+- [Geocoding](https://developers.arcgis.com/documentation/mapping-apis-and-services/geocoding/services/geocoding-service/) (**`arcgis`** support in development)
+- [Routing](https://developers.arcgis.com/documentation/mapping-apis-and-services/routing/services/routing-service/) (**`arcgis`** support in development)
 
-Josiah: we only support data hosting so we cannot mention any of these other options
-
--->
```

Comment:
I don't think this should be included right now. We only have partial support for "data hosting".

Can we add the comments back? 

For full data hosting support, we should be able to support vector tiles, map tiles, etc as well 
https://developers.arcgis.com/documentation/mapping-apis-and-services/data-hosting/feature-services/types-of-datasets/#feature-layers

Reply:
Will do! My interpretation was that simply linking to the ArcGIS Location Services was not implying that complete support was already implemented, but rather giving users an idea of the base functionality of those REST APIs. But fine to leave these bullets out until there is more built out in the packages- that makes sense.